### DISCUSSION TITLE
new menber

### DISCUSSION THREAD

**Discussion by darkvador123** [community/community#78126] [New to GitHub]
Hello, I am a new member on GitHub, and I am interested in computer systems and networks. I would like to learn and explore more with you. , i like too mobile phone , so if someone can to give me informations . thank  you.

**Comment by MuchaSsak** [comment]
Hi @darkvador123 !

Glad to have you here. If you need help getting started with Git and GitHub, you might want to watch a quick free crash course on YouTube or a different platform. 

A popular and a well received one is by freeCodeCamp - https://www.youtube.com/watch?v=RGOj5yH7evk
I recall watching a 30 minute video myself by TraversyMedia back in the day - https://www.youtube.com/watch?v=SWYqp7iY_Tc

If that's all you wanted to know, you may click `Mark as answer` down below.
If you need further help, you may reply to this comment.
